clients/lcdproc/mem.c

```diff
@@ -117,8 +117,11 @@ mem_screen(int rep, int display, int *flags_ptr)
 
 		/* flip the title back and forth... (every 4 updates) */
 		if (which_title & 4) {
-			sock_printf(sock, "widget_set M subtitle %i 1 {}\n", lcd_wid - 7);
-			sock_printf(sock, "widget_set M title {%s}\n", get_hostname());
+			if (strcmp(get_hostname(), "") != 0)
+			{
```

Comment:
Just check for get_hostname()[0] == 0, this is much simpler.

Also I wouldn't object, if you made hostname accessible everwhere and got rid of get_hostname() at all.

Reply:
Fixed. But i think sticking with get_hostname() is more nicer solution for me, i don't like shared variables between units.